src/controllers/controllermanager.h

```diff
@@ -58,6 +63,8 @@ class ControllerManager : public QObject {
     void requestShutdown();
     void requestSave(bool onlyActive);
     void requestInitialize();
+    void keyboardPresetChanged(KeyboardControllerPresetPointer pKbdPreset);
+    void keyboardEnabled(bool enabled);
 
   public slots:
     void updateControllerList();
```

Comment:
It looks like this is no slot anymore. 
Does it still do what it is named? It looks like you can update the List of connected controllers at any time. 
If so, all one time initializations must be moved out of this. Otherwise we could remove it from the slot list and rename it to InitControlerList or something.